testing/apps/details/Makefile

```diff
@@ -1,18 +1,15 @@
 APP_NAME 	:= $(shell basename `pwd`)
 IMAGE_NAME 	:= amalgam8/a8-testing-bookinfo-$(APP_NAME)
-LDFLAGS 	:=
 
-ifndef GOOS
-    GOOS := $(shell go env GOHOSTOS)
-endif
+BUILDFLAGS  := -i
+LDFLAGS 	:= -s -w -linkmode external -extldflags -static
 
-ifeq ($(GOOS),linux)
```

Comment:
I think this needs to change into GOHOSTOS . While the integration tests may run in Linux containers, the build may be on a Mac where the linker doesn't have the -s -w flags.